pkg/controller/command/didexchange/command.go

```diff
@@ -160,7 +161,7 @@ func (c *Command) GetHandlers() []command.Handler {
 		cmdutil.NewCommandHandler(commandName, createInvitationCommandMethod, c.CreateInvitation),
 		cmdutil.NewCommandHandler(commandName, receiveInvitationCommandMethod, c.ReceiveInvitation),
 		cmdutil.NewCommandHandler(commandName, acceptInvitationCommandMethod, c.AcceptInvitation),
-		cmdutil.NewCommandHandler(commandName, saveConnectionCommandMethod, c.SaveConnection),
+		cmdutil.NewCommandHandler(commandName, createConnectionCommandMethod, c.CreateConnection),
```

Comment:
We would need to update the wasm call - https://github.com/hyperledger/aries-framework-go/blob/016c620b82faa4c601ef183b421dc624dc2e3848/cmd/aries-js-worker/src/aries.js#L680

Reply:
@rolsonquadras fixed